eng/targets/VisualStudio.FastUpToDateCheckWorkarounds.targets

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project>
+
+  <!-- The 9.0 Arcade targets we are consuming don't have Sets specified for the items, and don't include all items correctly. This overrides
+       those targets so they don't interfere. These two empty targets can be deleted when we've removed the support from Arcade in favor of
```

Comment:
Can you link to the version of these targets we are currently referencing?

Reply:
It's this version:

https://github.com/dotnet/arcade/blob/bac7e1caea791275b7c3ccb4cb75fd6a04a26618/src/Microsoft.DotNet.Arcade.Sdk/tools/VisualStudio.VsixBuild.targets#L288-L310

I guess I could revise the comment, but fundamentally those need to be using the Set attribute or otherwise you end up with other issues if CoreCompile doesn't run. Which the targets in arcade main do.

Reply:
A newer feature: https://github.com/dotnet/project-system/blob/main/docs/up-to-date-check.md#grouping-inputs-and-outputs-into-sets